components/layout_2020/flow/inline/mod.rs

```diff
@@ -982,7 +971,7 @@ impl<'a, 'b> InlineFormattingContextState<'a, 'b> {
                 TextAlign::Start => text_indent,
                 TextAlign::End => (available_space - line_length).max(text_indent),
                 TextAlign::Center => {
-                    ((available_space - line_length + text_indent) / 2.).max(text_indent)
+                    ((available_space - line_length + text_indent) / 2).max(text_indent)
```

Comment:
Floating point division doesn't work how you would expect with Au, as it truncates. Instead you should use `.scale_by(...)`:

```suggestion
                    (available_space - line_length + text_indent).scale_by(0.5).max(text_indent)
```